### PR TITLE
Parse results from API to Post{} values

### PR DESCRIPTION
**Pull request by revett**
### Problem

Currently the results from the Facebook API are currently just sitting in `facebook.Result{}` objects.

### Solution

Parse the results into useable values in the `Post{}` struct.

### Notes

- https://trello.com/c/z7vx91W5/117-fbi-accessor-functions-for-post-struct-w-json-tags

### CODE REVIEW

File: post.go [hunk 1]
```diff
@@ -2,18 +2,34 @@ package fbintegration
...
-		ObjectID          string `facebook:"object_id" json:"object_id"`
-		Insights          *facebookLib.Result
-		ReactionBreakdown []*facebookLib.Result
-		TotalReactions    *facebookLib.Result
+		ID                 string `facebook:"id"        json:"post_id"`
+		ObjectID           string `facebook:"object_id" json:"object_id"`
+		Insights           *facebookLib.Result
+		ReactionBreakdown  []*facebookLib.Result
+		TotalReactions     *facebookLib.Result
+		Name               string         `json:"name"`
```
Comment (stevenjack) [inline post.go]: Might be worth nesting these into a struct that sits under `Insights` and the store the insight data in `Insights.Data`

🏄

**Comment by stevenjack** [comment]
:cake: looking good mate

**Comment by revett** [comment]
@stevenjack ready for another review

**Comment by stevenjack** [comment]
🍰  👌 🎉

**Pull request merged**